layouts/_default/baseof.html

```diff
@@ -42,115 +42,118 @@
-                <ul class="dropdown-menu">
-                  <li><a href="/mentoringprogramme.html">Mentoring Program</a></li>
-                  <li><a href="/mentorprogrammeformaleducation.html">Mentoring in Formal Education</a></li>
-                  <li><a href="/mentorprogrammeapplication.html">Applying for the Mentoring Program</a></li>
-                  <li class="divider"></li>
-                  <li class="nav-header">Additional Resources</li>
-                  <li><a href="/guide-to-being-a-mentor.html">Guide to being a Mentor</a></li>
-                  <li><a href="/mentor-request-mail.html">Mentor Request Email Example</a></li>
-                  <!-- <li><a href="/localmentors.html">Mentors Near You</a></li> -->
-                </ul>
-              </li>
-              <li class="dropdown">
-                <a href="#" class="dropdown-toggle" data-toggle="dropdown">Events <b class="caret"></b></a>
-                <ul class="dropdown-menu">
-				  <li><a href="https://apachecon.com/">ApacheCon - The official event of the ASF</a></li>
-                  <li><a href="/calendars/">Major Apache-related Events</a></li>
-                  <li><a href="https://www.apache.org/events/meetups.html">Upcoming Apache-related meetups</a></li>
-                  <li><a href="https://www.apache.org/foundation/marks/events">Apache event branding policies</a></li>
-                  <li class="divider"></li>
-                  <li class="nav-header">Speakers</li>
-                  <li><a href="/speakers/">Speaker Information</a></li>
-                  <li><a href="/speakers/speakers.html">Finding Speakers / Listing Yourself as a Speaker</a></li>
-                  <li><a href="/speakers/slides.html">Apache-related Slides</a></li>
-                </ul>
-              </li>
-              <li class="dropdown">
-                <a href="#" class="dropdown-toggle" data-toggle="dropdown">Join the Conversation<b class="caret"></b></a>
-                <ul class="dropdown-menu">
-                  <li><a href="https://www.facebook.com/ApacheSoftwareFoundation">Facebook</a></li>
-                  <li><a href="https://twitter.com/ApacheCommunity">Twitter</a></li>
-                  <li><a href="/lists.html">Mailing List</a></li>
-                </ul>
-              </li>
-            </ul>
-          </div><!--/.nav-collapse -->
-        </div>
-      </div>
+    <div class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
+      <!-- <a class="navbar-brand" href="#">Apache Community Development Project</a> -->
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#mainNavbar" aria-controls="mainNavbar" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="mainNavbar">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item"><a class="nav-link" href="#">Home</a></li>
```

Comment:
```suggestion
          <li class="nav-item"><a class="nav-link" href="/">Home</a></li>
```

Otherwise it won't navigate to the home page

Reply:
Fixed, thanks!